web3/packages/api-server/src/methods/index.ts

```diff
@@ -67,7 +57,6 @@ function getMethods(argsList: ModConstructorArgs = {}) {
               // express' error handler.
               //
               // Note: In order to link errors to transaction traces, we pass linking metadata.
```

Comment:
Shall we delete the related comments too?

Reply:
Updated